extensions/applicationinsights-clickanalytics-js/Tests/ClickEventTest.ts

```diff
@@ -0,0 +1,476 @@
+/**
+ * ClickEventTest.ts
+ * @author Krishna Yalamanchili (kryalama)
```

Comment:
This repo file headers doesn't include author and file name anywhere else, would be good to align with other files and simplify this